packages/pharaoh/lib/src/http/request.dart

```diff
@@ -71,12 +71,13 @@ class RequestContext {
 class Request extends Message<dynamic> implements $Request<dynamic> {
   final HttpRequest _req;
   final Map<String, dynamic> _params = {};
+  final Map<String, dynamic> _query = {};
   final Map<String, dynamic> _context = {};
 
   Request._(this._req) : super(_req, headers: {}) {
     req.headers.forEach((name, values) => headers[name] = values);
     headers.remove(HttpHeaders.transferEncodingHeader);
-    _params.addAll(_req.uri.queryParameters);
+    _query.addAll(_req.uri.queryParameters);
```

Comment:
See above.

Reply:
Totally understood! Lemme fix that real quick!